test-cases/performance/ycsb/perf-base.yaml

```diff
@@ -48,7 +48,7 @@ n_db_nodes: 3
 n_loaders: 1
 n_monitor_nodes: 1
 
-instance_type_db: 'i3.4xlarge'
+instance_type_db: 'i4i.4xlarge'
```

Comment:
This is a performance test, don't change it.

Reply:
Changed back.